js/runner.go

```diff
@@ -354,7 +358,7 @@ func (r *Runner) IsExecutable(name string) bool {
 }
 
 // HandleSummary calls the specified summary callback, if supplied.
-func (r *Runner) HandleSummary(ctx context.Context, summary *lib.Summary) (map[string]io.Reader, error) {
+func (r *Runner) HandleSummary(parentCtx context.Context, summary *lib.Summary) (map[string]io.Reader, error) {
```

Comment:
```suggestion
func (r *Runner) HandleSummary(ctx context.Context, summary *lib.Summary) (map[string]io.Reader, error) {
```

Why do you think is it better to rename it in this way? From a caller's point of view is it relevant? If yes, then we should document it. Otherwise, I think we should reverse the pattern and rename `s/ctx/summaryCtx/`.

Reply:
It is not better, but we need to add a sub-context inside of the method, so I chose to rename the parameter to `parentCtx`, to be able to create a `ctx` inside without overwriting this value.

Reply:
>but we need to add a sub-context inside of the method

I got that, I'm suggesting to reverse it selecting a specific name for the sub-context instead

Reply:
Resolved by the [latest force-push](https://github.com/grafana/k6/compare/9c9e35f63ef955140aff6ac122867832a249c7d8..a7ad53fef43a3a7885df96e08afab88c83d24436)